docs/developer.md

```diff
@@ -89,69 +62,23 @@ This is recommended when working on features that are purely Armada specific or
 
 #### Optional components
 
-##### NATS Streaming
-Armada can be set up to use NATS Streaming as message queue for events.
-To run NATS Streaming for development you can use docker:
-```bash
-docker run -d -p 4223:4223 -p 8223:8223 nats-streaming -p 4223 -m 8223
-```
-
-For Armada configuration check end to end test setup:
-```bash
-go run ./cmd/armada/main.go --config ./e2e/setup/insecure-armada-auth-config.yaml --config ./e2e/setup/nats/armada-config.yaml
-```
-
 ##### Lookout - Armada UI
-Lookout requires Armada to be configured with NATS Streaming.
-To run Lookout, firstly build frontend:
+Lookout requires Postgres to be up and running. Additionally, the react app must be built.
```

Comment:
Technically, Lookout requires both postgres and pulsar.  But I don't know if we really need this if someone runs the localdev script.  

Doesn't your script also build this react app?

Reply:
My script does not do the npm build -- you have to do the npm steps (right below this change). I guess I was being literal-minded about the dependencies (lookout-ingester needs pulsar, not lookout  - and NATS is no longer needed)